src/checks/index.ts

```diff
@@ -32,6 +32,11 @@ const linters = async () => {
             url: 'https://github.com/datenanfragen/data/blob/master/src/checks/pack',
             path_filter: (path: string) => path.startsWith('company-packs/'),
         },
+        generic: {
```

Comment:
```suggestion
        genericlint: {
```

Reply:
For consistency :D

Reply:
still unsure of the term "generic", but renamed :+1: